src/OpenLoco/src/Input/MouseInput.cpp

```diff
@@ -16,6 +16,7 @@
 #include "Objects/ObjectManager.h"
 #include "Objects/RoadObject.h"
 #include "SceneManager.h"
+#include "ToolManager.h"
```

Comment:
```suggestion
#include "Ui/ToolManager.h"
```

Reply:
No worries. I missed the other includes in this quick fix, though. Could you address the CI errors?